app/views/layouts/application.html.erb

```diff
@@ -1,4 +1,3 @@
-<% content_for :app_title, 'GOV.UK Signon' %>
 <% content_for :page_title, "#{yield(:title)} | GOV.UK Signon" %>
```

Comment:
At the moment there isn't a pattern for doing this automatically. But I think there should be.